<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QMainWindow" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calc BlackWood Peregrin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #3d3d3d;
font-family: &quot;Comic Sans MS&quot;;
</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>575</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="5" column="0">
      <widget class="QPushButton" name="but_7">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(254, 105, 193, 100);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="but_dot">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 23px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="but_0">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 23px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background:rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="but_1">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(139, 23, 167, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="but_9">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;	
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(139, 23, 167, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="but_2">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 23px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background:rgba(103, 10, 166, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_credit_term">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>Credit Term:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_cretidAmount">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: black;
color: white;
font-size: 25px;
font-family: &quot;Comic Sans MS&quot;;

</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verLayout_YM">
       <item>
        <widget class="QCheckBox" name="checkBox_years">
         <property name="styleSheet">
          <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
QCheckBox::indicator {
	color: rgba(90, 16, 180, 110);;
 }</string>
         </property>
         <property name="text">
          <string>Years</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_mounths">
         <property name="styleSheet">
          <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
QCheckBox::indicator {
	color: rgba(90, 16, 180, 110);;
 }</string>
         </property>
         <property name="text">
          <string>Mounths</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_cretidRate">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: black;
color: white;
font-size: 25px;
font-family: &quot;Comic Sans MS&quot;;

</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QVBoxLayout" name="verLayout_AD">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_annuity">
         <property name="styleSheet">
          <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
</string>
         </property>
         <property name="text">
          <string>Annuity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_diff">
         <property name="styleSheet">
          <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
QCheckBox::indicator {
	color: rgba(90, 16, 180, 110);;
 }</string>
         </property>
         <property name="text">
          <string>Differentiated</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="but_4">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(210, 73, 182, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="but_3">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 23px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="but_8">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(210, 73, 182, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_cretidTerm">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: black;
color: white;
font-size: 25px;
font-family: &quot;Comic Sans MS&quot;;

</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_rub">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>rub.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="but_5">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background: rgba(139, 23, 167, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_interest_rate">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>Interest Rate:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="but_6">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	width: 40px;
	height: 45px;
	border-radius: 23px;
	border: 3px solid rgb(12, 0, 0);
	font: bold 20px;
	color: white;
	background:rgba(103, 10, 166, 110);
	font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_credit_amount">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>Credit Amount:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_procent">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_payments">
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string>Type of Monthly Payment:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="but_calculate">
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 40px;
height: 45px;
border-radius: 23px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background:  rgba(60, 10, 200, 100);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>Ð¡alculate</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <layout class="QHBoxLayout" name="horLayout_but_clear">
       <item>
        <widget class="QPushButton" name="but_clear_last">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 30px;
height: 30px;
border-radius: 10px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>&lt;--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="but_clear">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
width: 30px;
height: 30px;
border-radius: 10px;
border: 3px solid rgb(12, 0, 0);
font: bold 20px;
color: white;
background: rgba(80, 16, 183, 110);
font-family: &quot;Comic Sans MS&quot;;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
